src/domain/community/user-preferences/dto/user-preference-definition.dto.create.ts

```diff
@@ -0,0 +1,9 @@
+import { UserPreferenceType, UserPreferenceValueType } from '@src/common';
```

Comment:
dangerous to import from the top level index file; will cause issues. Please import the types from teh actual folder.